.github/workflows/ci-pr-validation.yml

```diff
@@ -89,9 +89,9 @@ jobs:
 
       - name: Build Node binaries lib
         run: |
-          npm install --ignore-scripts
-          npx node-pre-gyp configure --target_arch=${{ matrix.arch }}
-          npx node-pre-gyp build --target_arch=${{ matrix.arch }}
+          yarn install --ignore-scripts
```

Comment:
We need to verify using both yarn and npm

Reply:
fixed.

Reply:
In fact, used npm install when building Windows as well as Linux Arch. This has verified that both yarn and npm can be used.

I'll keep using `yarn` when compiling macOS.